doc/conf.py

```diff
@@ -121,6 +121,8 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
+
+# NB: changes here should also be made to the linkcheck target in the Makefile
```

Comment:
Why do we want linkcheck to ignore our `_includes` folder? Isn't there valid rST in there that might contain links?

Reply:
Yes but it gets `.. include::`ed somewhere else. If you don't exclude it then you get hundreds of warnings about duplicated link names